_posts/2022-12-21-trino-summit-2022-astronomer-recap.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: post
+title: "Using Trino with Apache Airflow for (almost) all your data problems"
+author: "Philippe Gagnon, Brian Olsen"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-summit-2022/astronomer.jpg
+---
+
+As we close in on the final talks from [Trino Summit 2022]({% post_url 
+2022-11-21-trino-summit-2022-recap %}), this next talk dives into how to set up
+Trino for batch processing. Trino has historically been well-known for providing
+fast adhoc analytics queries as opposed to long-running, resource intensive
+queries commonly classified as batch or ETL queries. This is due to the fact
+that Trino kills queries that run out of resources in favor of lower latency.
```

Comment:
"in favor" doesn't make much sense because the context doesn't answer the question "in favor of what"

Suggest "...run out of resources in order to prioritize faster query execution" or similar